suites/pacific/rados/tier-2_rados_test-pool-functionalities.yaml

```diff
@@ -359,3 +359,13 @@ tests:
             compression_min_blob_size: 1B
             byte_size: 10KB
       desc: Verification of the effect of compression on replicated pools
+
+  - test:
+      name: Verify pool behaviour at min_size
+      module: pool_tests.py
+      polarion-id: CEPH-9185
+      config:
+        Verify_degraded_pool_min_size_behaviour:
+          pool_name: pool-degraded-test
+      desc: On a degraded cluster verify that clients can read and write data into pools with min_size OSDs
+      abort-on-fail: true
```

Comment:
If `abort-on-fail` is set to true, we should probably keep this test at the very bottom so that succeeding tests do not get affected. If we decide to not do that, we can ensure the test case cleanup is appropriate so that following tests get a clean cluster to begin with.
This is open to discussion